mobile-app/app/screens/AppNavigator/screens/Loans/components/EmptyVault.tsx

```diff
@@ -68,6 +69,14 @@ export function EmptyVault (props: EmptyVaultProps): JSX.Element {
           text='Learn more about vaults'
           testId='empty_vault_learn_more'
         />
+        <TouchableOpacity onPress={() => navigation.navigate('LoansTnC')}>
+          <InfoText
+            simple
+            text={translate('components/LoansFaq', 'Loan terms')}
+            style={tailwind('p-2 flex-row items-end justify-start')}
+          // testId='empty_vault_learn_more'
```

Comment:
what is this testId? is it used for unit & integration testing? why is it commented out?

Reply:
most probably for both of 'em